src/components/icons/Rake.tsx

```diff
@@ -1,52 +1,67 @@
 export const Rake = () => (
-  <svg
-    width="32"
-    height="33"
-    viewBox="0 0 32 33"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
-  >
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M3 20.3799C3.55228 20.3799 4 20.8276 4 21.3799V26.3799C4 26.9322 3.55228 27.3799 3 27.3799C2.44772 27.3799 2 26.9322 2 26.3799V21.3799C2 20.8276 2.44772 20.3799 3 20.3799Z"
-      fill="black"
-    />
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M7 20.8799C7.55228 20.8799 8 21.3276 8 21.8799V26.8799C8 27.4322 7.55228 27.8799 7 27.8799C6.44772 27.8799 6 27.4322 6 26.8799V21.8799C6 21.3276 6.44772 20.8799 7 20.8799Z"
-      fill="black"
-    />
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M11 21.3799C11.5523 21.3799 12 21.8276 12 22.3799V27.3799C12 27.9322 11.5523 28.3799 11 28.3799C10.4477 28.3799 10 27.9322 10 27.3799V22.3799C10 21.8276 10.4477 21.3799 11 21.3799Z"
-      fill="black"
-    />
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M10.3121 23.1052C9.9112 22.7254 9.89409 22.0925 10.2739 21.6915L28.2739 2.69153C28.6538 2.29059 29.2867 2.27349 29.6876 2.65332C30.0886 3.03315 30.1057 3.66608 29.7258 4.06702L11.7258 23.067C11.346 23.4679 10.7131 23.4851 10.3121 23.1052Z"
-      fill="black"
-    />
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M15 21.8799C15.5523 21.8799 16 22.3276 16 22.8799V27.8799C16 28.4322 15.5523 28.8799 15 28.8799C14.4477 28.8799 14 28.4322 14 27.8799V22.8799C14 22.3276 14.4477 21.8799 15 21.8799Z"
-      fill="black"
-    />
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M19 22.3799C19.5523 22.3799 20 22.8276 20 23.3799V28.3799C20 28.9322 19.5523 29.3799 19 29.3799C18.4477 29.3799 18 28.9322 18 28.3799V23.3799C18 22.8276 18.4477 22.3799 19 22.3799Z"
-      fill="black"
-    />
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M2.00783 21.256C2.07633 20.7079 2.57612 20.3192 3.12414 20.3877L19.1241 22.3877C19.6722 22.4562 20.0609 22.956 19.9924 23.504C19.9239 24.052 19.4241 24.4408 18.8761 24.3723L2.87607 22.3723C2.32805 22.3038 1.93932 21.804 2.00783 21.256Z"
-      fill="black"
-    />
-  </svg>
+  <>
+    <style>
+      {`
+        .rake-icon {
+          fill: black;
```

Comment:
We don't want to explicitly set the colors based on the color scheme because WHCM users can set their own colors and this would override that. You can instead use the system colors, so in this example it would be `CanvasText` for regular text.

More info: https://blogs.windows.com/msedgedev/2020/09/17/styling-for-windows-high-contrast-with-new-standards-for-forced-colors/